build_runner/lib/src/entrypoint/base_command.dart

```diff
@@ -88,7 +88,11 @@ abstract class BuildRunnerCommand extends Command<int> {
       ..addFlag(symlinkOption,
           defaultsTo: symlinksDefault,
           negatable: true,
-          help: 'Symlink files in the output directories, instead of copying.');
+          help: 'Symlink files in the output directories, instead of copying.')
+      ..addMultiOption(buildFilterOption,
+          help: 'An explicit filter of files to build. Relative paths and '
```

Comment:
We need to make it clear that this is an `OR` not an `AND` when you give multiple filters.